sherpa/ui/tests/test_session.py

```diff
@@ -21,6 +21,13 @@
 from numpy.testing import assert_array_equal
 
 
+# bug #303
+def test_set_log():
```

Comment:
If you did want to make these tests a bit more functional, you can check the setting of the get_*_plot_prefs dictionary before and after the call - e.g. as shown below:

```
>>> from sherpa.astro.ui.utils import Session
>>> session = Session()
>>> session.get_data_plot_prefs()['xlog']
False
>>> session.get_data_plot_prefs()['ylog']
False
>>> session.set_xlog()
>>> session.get_data_plot_prefs()['xlog']
True
>>> session.get_data_plot_prefs()['ylog']
False
>>> session.set_ylog('model')
>>> session.get_data_plot_prefs()['ylog']
False
>>> session.get_model_plot_prefs()['ylog']
True
>>> session.set_ylog()
>>> session.get_data_plot_prefs()['ylog']
True
>>> session.get_model_plot_prefs()['ylog']
True
>>> session.set_ylinear()
>>> session.get_data_plot_prefs()['ylog']
False
>>> session.get_model_plot_prefs()['ylog']
False
```
I'll see if I can come up with a simple example test (or two).